src/System.ComponentModel.TypeConverter/tests/Drawing/ImageConverterTests.cs

```diff
@@ -0,0 +1,354 @@
+        public void ConvertTo_Int()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConv.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(int)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Rectangle()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(Rectangle)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Image()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, _image.GetType()));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Size()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(Size)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Bitmap()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(Bitmap)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Point()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(Point)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Metafile()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(Metafile)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Object()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(object)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertTo_Int()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(int)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TestGetPropertiesSupported()
+        {
+            Assert.True(_imgConv.GetPropertiesSupported(), "GPS#1");
+            Assert.True(_imgConv.GetPropertiesSupported(null), "GPS#2");
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TestGetProperties()
+        {
+            int basecount = 1;
+            PropertyDescriptorCollection propsColl;
+
+            propsColl = _imgConv.GetProperties(null, _image, null);
+            Assert.Equal(13 + basecount, propsColl.Count);
```

Comment:
Could you include a comment of why it is 13 + basecount and 6 + basecount the expected values?